v2/spiffetls/tlsconfig/hooks.go

```diff
@@ -0,0 +1,10 @@
+package tlsconfig
+
+import (
+	"github.com/spiffe/go-spiffe/v2/spiffeid"
+)
+
+type Hooks struct {
```

Comment:
How would you feel about another name, like `Trace` or something? `Hooks`, at least to me, implies that the behavior can be modified by the hook. Maybe I'm alone in thinking that though :)

Reply:
I don't have a preference on the name, `Trace` is fine to me and conveys the intent better. :+1:

Reply:
Renamed to trace.